pyaedt/hfss.py

```diff
@@ -4561,6 +4565,49 @@ def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"
             bound.props["Type"] = hybrid_region
         return bound
 
+    @pyaedt_function_handler()
+    def assign_febi(self, obj_names, boundary_name=""):
+        """Assign an FE-BI region to one or more objects.
+
+        Parameters
+        ----------
+        obj_names : str or list or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            One or more object names or IDs.
+        boundary_name : str, optional
+            Name of the boundary. The default is ``""``.
+
+        Returns
+        -------
+        :class:`pyaedt.modules.Boundary.BoundaryObject`
+            Boundary object.
+
+        References
+        ----------
+
+        >>> oModule.AssignFEBI
+
+        Examples
+        --------
+
+        Create a box and assign an FEBI boundary to it.
```

Comment:
```suggestion
        Create a box and assign an FE-BI boundary to it.
```
Check for consistency. It looks like FE-BI is preferred.